classes/block_manager.php

```diff
@@ -361,14 +360,18 @@ public function edit_controls($block) {
         if (empty($blocktitle)) {
             $blocktitle = $block->arialabel;
         }
+        $blockregion = $block->instance->region;
```

Comment:
are we 100% sure that $block->instance->region is always set? If not, maybe better this: 

`$blockregion = !empty($block->instance->region) ? $block->instance->region : $block->instance->defaultregion;`

        if (empty($blockregion)) {
            $blockregion = $block->instance->defaultregion;
        }

Reply:
It checks if $blockregion is empty just below, but this is tidier